### PR TITLE
LifeCycleAssessment_oM: Modification of MEP Scope Object

### LINKED ISSUE #1043: LifeCycleAssessment_oM: Modification of MEP Scope Object
#### Description:
MEP scope object to include holistic MEP assessment (addition of several categories)

### PR DESCRIPTION
**Pull request by kayleighhoude**
Addition of MEP scope object inputs. Inputs are in order of perceived importance/weight for LCA related factors.

Additional objects input allows for objects that potentially won't be BHoM specific types (eg filters, hangers, brackets)

### Issues addressed by this PR

Closes #1043 

### Test files

image

### CODE REVIEW

File: LifeCycleAssessment_oM/ScopeObjects/MechanicalScope.cs [hunk 1]
```diff
@@ -33,16 +33,31 @@ public class MechanicalScope : BHoMObject
         /***************************************************/
         /**** Properties                                ****/
         /***************************************************/
-        [Description("Mechanical distribution system that convey airflow from heating, ventilation or cooling systems (e.g. sheet metal)")]
-        public virtual List<IElementM> Ducts { get; set; } = new List<IElementM>();
```
Comment (michaelhoehn) [inline LifeCycleAssessment_oM/ScopeObjects/MechanicalScope.cs:37]: Alphabetical, please.

File: LifeCycleAssessment_oM/ScopeObjects/PlumbingScope.cs [hunk 2]
```diff
@@ -33,12 +33,21 @@ public class PlumbingScope : BHoMObject
...
-        [Description("Plumbing equipment includes machines or devices that pump, store or process water, waste and sewage streams, etc.")]
+        [Description("Plumbing equipment includes machines or devices that pump, store or process water, gas or waste.")]
         public virtual List<IElementM> Equipment { get; set; } = new List<IElementM>();
         
-        [Description("Plumbing pipework includes distribution systems (e.g. PVC) that convey fluids")]
+        [Description("Plumbing pipework includes distribution systems that convey fluids (e.g. domestic cold water, hot water, gas, vent, waste).")]
         public virtual List<IElementM> Pipes { get; set; } = new List<IElementM>();
 
+        [Description("Valves are devices that control the flow or pressure within a plumbing piping system (e.g. ball valve, globe valve, gate valve).")]
+        public virtual List<IElementM> Valves { get; set; } = new List<IElementM>();
```
Comment (michaelhoehn) [inline LifeCycleAssessment_oM/ScopeObjects/PlumbingScope.cs:43]: Alphabetical, please.

**Review by michaelhoehn — CHANGES_REQUESTED** [review]
Minor comments for organisation that are helpful for dependent methods.

File: LifeCycleAssessment_oM/ScopeObjects/ElectricalScope.cs [hunk 3]
```diff
@@ -33,26 +33,47 @@ public class ElectricalScope : BHoMObject
         /***************************************************/
         /**** Properties                                ****/
         /***************************************************/
+        [Description("Electrical Equipment includes machines or devices that comprise the electrical infrastructure (e.g. switchgear, panels, transformers, automatic transfer switches).")]
```
[the remark this answers is not in the document]
Comment (michaelhoehn) [reply LifeCycleAssessment_oM/ScopeObjects/ElectricalScope.cs:36]: Also, AdditionalObjects can remain at the bottom as a deliberate UI decision that is common across all scopes.

**Review by FraserGreenroyd — APPROVED** [review]
Code LGTM

**Review by michaelhoehn — APPROVED** [review]

**Pull request merged**